src/app/core/admin/admin-entity-details/widget-component-select/widget-component-select.component.ts

```diff
@@ -29,17 +30,111 @@ import { MatTooltipModule } from "@angular/material/tooltip";
 export class WidgetComponentSelectComponent implements OnInit {
   private entityRelationsService = inject(EntityRelationsService);
   private dialogRef =
-    inject<MatDialogRef<WidgetComponentSelectComponent, PanelComponent>>(
-      MatDialogRef,
-    );
+    inject<MatDialogRef<WidgetComponentSelectComponent, any>>(MatDialogRef);
+
   data = inject<{
-    entityType: string;
+    entityType?: string;
+    isDashboard?: boolean;
   }>(MAT_DIALOG_DATA);
 
-  options: WidgetOption[];
+  options: WidgetOption[] | DashboardWidgetOption[];
 
   ngOnInit() {
-    this.options = this.loadAvailableWidgets();
+    if (this.data.isDashboard) {
+      this.options = this.loadDashboardWidgets();
+    } else {
+      this.options = this.loadAvailableWidgets();
+    }
+  }
+
+  private loadDashboardWidgets(): DashboardWidgetOption[] {
+    return [
+      {
+        label: $localize`Shortcuts`,
+        value: {
+          component: "ShortcutDashboard",
+          config: {
+            shortcuts: [
+              {
+                label: "Record Attendance",
+                icon: "calendar-check",
+                link: "/attendance/add-day",
+              },
+              {
+                label: "Add Child",
+                icon: "plus",
+                link: "/child/new",
+              },
+              {
+                label: "Public Registration Form",
+                icon: "file-circle-check",
+                link: "/public-form/form/test",
+              },
+            ],
+          },
+        },
+      },
+      {
+        label: $localize`Entity Count`,
+        value: {
+          component: "EntityCountDashboard",
+          config: {},
+        },
+      },
+      {
+        label: $localize`Important Notes`,
+        value: {
+          component: "ImportantNotesDashboard",
+          config: { warningLevels: ["WARNING", "URGENT"] },
+        },
+      },
+      {
+        label: $localize`Todos`,
+        value: {
+          component: "TodosDashboard",
+          config: {},
+        },
+      },
+      {
+        label: $localize`Notes`,
+        value: {
+          component: "NotesDashboard",
+          config: {
+            sinceDays: 28,
+            fromBeginningOfWeek: false,
+            mode: "with-recent-notes",
+          },
+        },
+      },
+      {
+        label: $localize`Attendance (This Week)`,
+        value: {
+          component: "AttendanceWeekDashboard",
+          config: { daysOffset: 7, periodLabel: "this week" },
+        },
+      },
+      {
+        label: $localize`Attendance (Last Week)`,
+        value: {
+          component: "AttendanceWeekDashboard",
+          config: { daysOffset: 0, periodLabel: "last week" },
+        },
+      },
+      {
+        label: $localize`Progress`,
+        value: {
+          component: "ProgressDashboard",
+          config: { dashboardConfigId: "1" },
```

Comment:
TODO: needs to generate a new ID, otherwise the progress states will always be exactly the same between multiple widgets of this type

[--> Pair Programming Session]